ci/pipeline

```diff
@@ -199,7 +214,7 @@ def pr(): Unit = asPullRequest {
   val version = build()
 
   // Uploads
-  val artifact = uploadTarballPackagesToS3(version)
+  val artifact = uploadTarballPackagesToS3(version, "snapshots")
```

Comment:
I have to read up on the doc but should this be `builds` as well? I guess not.

Reply:
same as above... and yeah.